server/graphql/types.js

```diff
@@ -70,10 +72,14 @@ const getCustomTypes = (strapi, nexus) => {
 							},
 						};
 
-						// only return published entries
+						// return entries based on publicationState arg or default to only returning published entries when draftAndPublish mode is enabled
 						if (_.get(contentType, ['options', 'draftAndPublish'], false)) {
-							query.publicationState = 'live';
-						}
+							query.publicationState = "live";
+							
+							if (publicationState) {
+								query.publicationState = publicationState;
+							}
+						} 
```

Comment:
We should be able to use a shorted boolean check here.
```suggestion
							query.publicationState = publicationState || 'live';
						} 
```